tests/unit_tests/db_engine_specs/test_presto.py

```diff
@@ -46,9 +53,32 @@
 def test_convert_dttm(
     target_type: str,
     dttm: datetime,
-    result: Optional[str],
+    expected_result: Optional[str],
+) -> None:
+    from superset.db_engine_specs.presto import PrestoEngineSpec as spec
+
+    assert_convert_dttm(spec, target_type, expected_result, dttm)
+
+
+@pytest.mark.parametrize(
+    "native_type,sqla_type,attrs,generic_type,is_dttm",
+    [
+        ("varchar(255)", types.VARCHAR, {"length": 255}, GenericDataType.STRING, False),
+        ("varchar", types.String, None, GenericDataType.STRING, False),
+        ("char(255)", types.CHAR, {"length": 255}, GenericDataType.STRING, False),
+        ("char", types.String, None, GenericDataType.STRING, False),
+        ("integer", types.Integer, None, GenericDataType.NUMERIC, False),
+        ("time", types.Time, None, GenericDataType.TEMPORAL, True),
+        ("timestamp", types.TIMESTAMP, None, GenericDataType.TEMPORAL, True),
```

Comment:
`date` etc? do these mean superset will only support these Presto types?
prob a silly question but do these come from: https://github.com/dropbox/PyHive/blob/master/pyhive/sqlalchemy_presto.py#L30?

Reply:
I only ported over the current tests from the old `presto_tests.py` which apparently didn't have a test for `DATE`. However, notice that we're testing for it in `convert_dttm` above, so it's working at least. Once this is merged I'm hoping I can do a few more rounds of improving coverage.